src/core/evaluator.js

```diff
@@ -2352,6 +2350,10 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           }
         }
 
+        const firstChar = (dict.get('FirstChar') || 0);
+        const lastChar = (dict.get('LastChar') || (composite ? 0xFFFF : 0xFF));
+        hash.update(`${firstChar}-${lastChar}`);
```

Comment:
The updated patch only changed the separator here, since using the same one (i.e. comma) as in the `'Differences' array` case above may perhaps not be the best idea.